auditbeat/module/file_integrity/event.go

```diff
@@ -182,6 +184,16 @@ func NewEventFromFileInfo(
 		return event
 	}
 
+	if len(fileParsers) != 0 && event.ParserResults == nil {
+		event.ParserResults = make(common.MapStr)
+	}
+	for _, p := range fileParsers {
+		err = p.Parse(event.ParserResults, path)
+		if err != nil {
+			event.errors = append(event.errors, err)
+		}
+	}
+
```

Comment:
This should probably have INFO level total work time logging and per processor DEBUG level work time logging.